aztec/src/main/kotlin/org/wordpress/aztec/formatting/InlineFormatter.kt

```diff
@@ -73,6 +74,14 @@ class InlineFormatter(editor: AztecText, val codeStyle: CodeStyle) : AztecFormat
                     AztecTextFormat.FORMAT_CODE -> {
                         applyInlineStyle(item, textChangedEvent.inputStart, textChangedEvent.inputEnd)
                     }
+                    AztecTextFormat.FORMAT_MARK -> {
+                        // For cases of an empty mark tag, either at the beginning of the text or in between
+                        if (textChangedEvent.inputStart == 0 && textChangedEvent.inputEnd == 1) {
+                                applyMarkInlineStyle(textChangedEvent.inputStart, textChangedEvent.inputEnd)
```

Comment:
I think there is extra indentation on this line though the linter did not complain 🤔